src/main/java/hw11and12/FileManagerIO.java

```diff
@@ -0,0 +1,94 @@
+package hw11and12;
+
+import java.io.*;
+
+public class FileManagerIO {
+    private String currentDirectory;
+    public void run() throws IOException {
+
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        currentDirectory = System.getProperty("user.dir");
+        while (true) {
+            System.out.print(currentDirectory + ">");
```

Comment:
Instead of using print statements, use `java.util.Logger` to print the output you need